examples/favorites_advanced/getx_favorites_advanced/lib/base/resources/input_styles.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
```

Comment:
@RomanovaPrime, can you please explain why we need input styles on the project level?

Reply:
Resolved in another branch.